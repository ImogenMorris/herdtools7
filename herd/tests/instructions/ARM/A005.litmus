ARM A005

Stable=R1,R2,R4,R5

(*Test LDRD Instruction *)
{ int x[2] = {1,1}; 0:R3 = x}

P0;
LDRD R1,R2,[R3];
LDRD R4,[R3];

forall (0:R1=1 /\ 0:R2=1 /\ 0:R4=1 /\ 0:R5=1)
