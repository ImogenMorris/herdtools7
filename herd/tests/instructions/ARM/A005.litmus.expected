Test A005 Required
States 1
0:R2=1; 0:R3=1; 0:R4=1; 0:R5=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:R2=1 /\ 0:R3=1 /\ 0:R4=1 /\ 0:R5=1)
Observation A005 Always 1 0
Hash=afad8df5dbf04b59b889b889ad2b7285

